pkg/external-boot-image/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# syntax=docker/dockerfile-upstream:1.5.0-rc2-labs
+
+FROM lfedge/eve-xen-tools:e36677ad441c2d5cd88cdd9831959eaa4715ab82 as initrd-build
+FROM docker.io/lfedge/eve-kernel:eve-kernel-amd64-v6.1.38-generic-d2da815271a2-gcc as  kernel-build
+
+FROM scratch
+COPY --from=initrd-build --chmod=666 /usr/lib/xen/boot/runx-initrd /
+COPY --from=kernel-build --chmod=666 /kernel /
```

Comment:
@zedi-pramodh , I realized that you forgot to copy the kernel modules, you are copying only the kernel and initrd images, you need to copy the modules as well...

Reply:
Actually kubevirt only takes kernel and initrd.   Look at  https://kubevirt.io/user-guide/virtual_machines/boot_from_external_source/

Reply:
ok, but in this case you will need to handle the modules anyways, otherwise the VM might not work properly... maybe create a new initrd and put the modules inside... or does kubevirt handles modules somehow outside this container?

Reply:
I have to guess so, because VMs are running fine and ztests passed. I also logged into VMs and verified things are fine.

Reply:
It makes sense because you are using our initrd (from pkg/xen-tools), which mounts the module dir to the container, but at the other hand I did't understand where and how exactly this container is initialized.... but you said it's working fine, so have you checked the modules, they are there (under /lib/modules/....), right?

Reply:
This container is initialized in domainmgr code. We pass it to kubevirt in the spec. You will see that code in future commits. But the idea is described in the URL I posted earlier.

https://kubevirt.io/user-guide/virtual_machines/boot_from_external_source/